modelconverter/hub/__main__.py

```diff
@@ -1,12 +1,17 @@
 import logging
+import webbrowser
 from pathlib import Path
-from typing import Any, Dict, List, Literal, Optional
+from time import sleep
+from typing import Any, Dict, List, Optional
 from urllib.parse import unquote, urlparse
 
+import click
+import keyring
```

Comment:
Do these 2 need to be added to requirements?

Reply:
Only `keyring` byt yes, thanks